gdb2cm.go

```diff
@@ -53,33 +56,58 @@ func readDashboardJson(file string) *models.Dashboard {
 	return dbo
 }
 
-func main() {
-	kingpin.Parse()
-	if !strings.HasSuffix(*dashboardFile, ".json") {
-		panic(fmt.Sprintf("%s is not a file...exiting", *dashboardFile))
+func processDir(dbDirGlob, mDir string) {
```

Comment:
`processDirs`. Can't decide if you should have a `processDir` that is called by the `processDirs`